jaeger-core/src/test/java/io/jaegertracing/internal/JaegerObjectFactoryTest.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2018, Uber Technologies, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.jaegertracing.internal;
+
+import io.jaegertracing.Configuration;
+import io.opentracing.Scope;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JaegerObjectFactoryTest {
```

Comment:
technically, this is not testing the object factory, but the extension mechanism.

Reply:
Fair enough. Will revert renaming.